packages/ui/core-components/src/lib/unsorted/ui/Footnote.svelte

```diff
@@ -0,0 +1,18 @@
+<script context="module">
```

Comment:
There is some semi standard[^1] syntax for footnotes in markdown. I'd probably be keen to support this via a remark plugin instead, as it deals with linking the notes also.

[^1]: For example GFM, but it's not in the CommonMark spec

Reply:
https://github.com/remarkjs/remark-gfm

Reply:
Agree that's probably better - will take a look, but if it's a time sink I think this rough version is a good thing to have in the meantime